packages/landing/src/components/map.tsx

```diff
@@ -80,14 +81,82 @@ export class Basemaps extends Component<unknown, { isLayerSwitcherEnabled: boole
   updateStyle = (): void => {
     this.ensureGeoControl();
     const tileGrid = getTileGrid(Config.map.tileMatrix.identifier);
-    const style = tileGrid.getStyle(Config.map.layerId, Config.map.style);
+    const style = tileGrid.getStyle(Config.map.layerId, Config.map.style, undefined, Config.map.dateRange);
     this.map.setStyle(style);
 
     if (Config.map.tileMatrix !== GoogleTms) this.map.setMaxBounds([-180, -85.06, 180, 85.06]);
     else this.map.setMaxBounds();
+    // TODO check and only update when Config.map.layer changes.
```

Comment:
I think we can check layers changes from Attribution now, we don't need Config.map.layer at here anymore.